app/views/shared/_header.html.erb

```diff
@@ -4,10 +4,14 @@
       <div class="small-12 medium-6 column <%= "medium-9 small-centered text-center" unless header.image.present? %>">
         <span><%= header.label %></span>
         <h1><%= header.title %></h1>
+
         <p class="lead"><%= header.description %></p>
-        <div class="small-12 medium-6 large-4 <%= "small-centered" unless header.image.present? %>">
-          <%= link_to header.link_text, header.link_url, class: "button expanded large" %>
-        </div>
+
+        <% if header.link_text.present? %>
+          <div class="link-content small-12 medium-6 large-4 <%= "small-centered" unless header.image.present? %>">
```

Comment:
We can remove the `link-content` class if we apply the changes mentioned in the tests.